risc0/zkp/core/Cargo.toml

```diff
@@ -1,13 +1,25 @@
 [package]
 name = "risc0-zkp-core"
-version = "0.1.0"
+version = "0.2.0"
 edition = "2021"
 description = "RISC Zero zero-knowledge proof system core crate"
 license = "Apache-2.0"
 homepage = "https://risczero.com/"
 repository = "https://github.com/risc0/risc0/"
+links = "risc0-zkp-core"
+
+[features]
+default = ["pure"]
```

Comment:
Can we document what this "pure" feature does somewhere?

Reply:
Maybe I should call it the `rust` feature? Where should I document this?

Reply:
I added some comments to this file.